components/search/MenuBar.jsx

```diff
@@ -0,0 +1,93 @@
+import {useContext, useState} from 'react'
+// import dynamic from 'next/dynamic'
+import  { ScrollMenu, VisibilityContext } from "react-horizontal-scrolling-menu";
+
+
+
+
+function MenuBar() {
+  const [items, setItems] = useState([
+    {
+    title:'food',
+    id: 0,
+  }
+  ,{
+    title:'sports',
+    id: 1,
+  }
+  ,{
+    title:'movies',
+    id: 2,
+  }
+  ,{
+    title:'politics',
+    id: 3,
+  }
+  ,{
+    title:'test',
+    id: 4,
+  }
+  ,{
+    title:'test',
+    id: 5,
+  }
+]);
+  const [selected, setSelected] = useState([]);
+  const [position, setPosition] = useState(0);
+
+  const isItemSelected = (id) => !!selected.find((el) => el === id);
+
+  const handleClick = (id) => ({ getItemById, scrollToItem }) => {
+    const itemSelected = isItemSelected(id)
+
+    setSelected((currentSelected) =>
+      itemSelected
+        ? currentSelected.filter((el) => el !== id)
+        : currentSelected.concat(id)
+    );
+  }
+
+  return (
+        <ScrollMenu >
+          {items.map(({ id, title }) => (
+            <Card
+              itemId={id} // NOTE: itemId is required for track items
+              title={title}
+              key={id}
+              onClick={handleClick(id)}
```

Comment:
This click event is not triggering in the browser. It is not throwing any errors either.